### PR TITLE
Modified e2e regression tests to reproduce only one bug

### PR DESCRIPTION
**Pull request by ThrivikramanV**
The e2e regression tests on every PR were reproducing all the bugs and hence consuming a lot of time. Have modified it to reproduce only one bug (chosen at random).

Screenshot from 2023-10-31 19-50-16

### CODE REVIEW

**Comment by tylergu** [comment]
The change looks good, just a minor comment on naming the tests, as the first step to make the per-PR bug reproduction CI work again. I think we should explicitly name the tests as bug reproduction tests, because there are other regression tests in the repo that are not doing bug reproduction.

We will add a weekly action to reproduce all bugs after this.

File: pytest.ini [hunk 1]
```diff
@@ -1,3 +1,4 @@
 [pytest]
 markers =
-    local: mark a test to run on a local machine
\ No newline at end of file
+    local: mark a test to run on a local machine
+    regression: mark a test to run e2e regressions on a pull request
```
[the remark this answers is not in the document]
Comment (ThrivikramanV) [reply pytest.ini]: Done

**Comment by tylergu** [comment]
I think the `Regression Test / unittest (push) ` action is also running the bug reproduction, because of how the pytest mark is specified in its action file. We may need a good way to distinguish between unittest and end-to-end tests

**Comment by tylergu** [comment]
@ThrivikramanV Do you know what is causing the e2e test to be canceled?

**Comment by ThrivikramanV** [comment]
I had manually canceled it since I assumed that it would be running the original regressions (reproducing all the bugs) until we merge this PR with main. Is this not the case?

**Comment by tylergu** [comment]
I think it would run against the PR branch, which contains the updated github actions.
The only thing to fix is the `Regression Test / unittest (push)` I think

**Pull request merged**